boxwise_flask/routes.py

```diff
@@ -79,7 +79,7 @@ def graphql_server():
     # Note: Passing the request to the context is optional.
     # In Flask, the current request is always accessible as flask.request
 
-    debug_graphql = bool(os.getenv("DEBUG_GRAPHQL", False))
+    debug_graphql = True  # bool(os.getenv("DEBUG_GRAPHQL", False))
```

Comment:
why?

Reply:
My bad this is still a draft. Graphql gives little information when models don't line up correctly. just wanted to make sure I was getting all the information